tests/status/test_legacy_tracker.py

```diff
@@ -89,7 +89,7 @@ def check_expression(original, path_expression, expected, msg_start):
             "poly_example",
             [
                 ENSEMBLE_SMOOTHER_MODE,
-                "--disable-ensemble-evaluator",
+                "--enable-ensemble-evaluator",
```

Comment:
I have yet to figure out why, but it seems like this test is leaking into the `failing_poly_example`. If I remove all the other test cases but `failing_poly_example` it passes. If I reintroduce all cases but this one it passes (but the top one stil fails). If I reintroduce this case as well, `failing_poly_example` starts failing. If I revert this case also to run without the ensemble evaluator, `failing_poly_example` starts passing again.

It seems like mixing tests using and not using the ensemble evaluator does not mix well 🤔